addons/sale/models/sale.py

```diff
@@ -1363,6 +1363,8 @@ def _compute_untaxed_amount_to_invoice(self):
                     price_subtotal = line.price_reduce * line.qty_delivered
                 else:
                     price_subtotal = line.price_reduce * line.product_uom_qty
+                if len(line.tax_id.filtered(lambda tax: tax.price_include)) > 0:
```

Comment:
Maybe add an extra comment to explain why we are doing this.

Reply:
Done.